src/utils/trusts.ts

```diff
@@ -446,6 +445,16 @@ function getRoleWithinTrustType(type: any): RoleWithinTrustType | undefined {
   return undefined;
 }
 
+function convertBooleanToYesNoResponse(apiYesNoResponse: yesNoResponse): yesNoResponse {
+  // used to convert boolean recieved from api to number for yesNoResponse used in web.
+  // although true/false is sent by the api, typescript still detects it as a yesNoResponse hence why yesNoResponse is passed to this function
+
+  if (apiYesNoResponse === undefined){
+    return yesNoResponse.No;
+  }
+  return Number(apiYesNoResponse);
```

Comment:
parseInt may be better here as you are expecting an int. Nonetheless will still work.

Reply:
vscode complains when using parseInt in this case